docs/databases/hadoop/how-to-install-and-set-up-hadoop-cluster/index.md

```diff
@@ -77,21 +79,23 @@ For each node to communicate with each other by name, edit the `/etc/hosts` file
 
 ### Distribute Authentication Key-pairs for the Hadoop User
 
-The master node will use an ssh-connection to connect to other nodes with key-pair authentication, to manage the cluster.
+The master node will use an ssh connection to connect to other nodes with key-pair authentication. This will allow the master node to actively manage the cluster.
 
 1.  Login to **node-master** as the `hadoop` user, and generate an ssh-key:
 
         ssh-keygen -b 4096
 
-1.  View the **node-master** public key so you can copy it to each of the worker nodes.
+     When generating this key, leave the password field blank so your hadoop user can communicate unprompted.
+
+1.  View the **node-master** public key and copy it to your clipboard to use with each of your worker nodes.
```

Comment:
felt this clarification was worth it for people who may not fully understand how `less` works